MAINTAINERS.md

```diff
@@ -138,3 +138,10 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
 - Leandro Ostera ([@ostera](https://github.com/ostera)):
   until [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454)
 - Waldir Pimenta ([@waldyrious](https://github.com/waldyrious)): until [26 August 2018](https://github.com/tldr-pages/tldr/issues/2257)
+
+## Past organization members
+
+Organization members who have been inactive for some time, but still have write access to all the repositories in the tldr-pages organization and are welcome back any time to return to their old role.
```

Comment:
```suggestion
Organization members who have been inactive for some time, still have write access to all the repositories in the tldr-pages organization, after losing organization status, previous members are welcome back any time to return to their old role.
```

Reply:
Let's reserve this type of change for a different PR.  I have a PR #5770 that can touch this file as well.  (Also, `losing` is really too negative, I think.)

Reply:
```suggestion
Organization members who have been inactive for some time, still have write access to all the repositories in the tldr-pages organization. After releasing organization status, previous members are welcome back any time to return to their old role.
```

Reply:
How about just "removing"?

Reply:
releasing is less _negative_ than removing

Reply:
True, so much so that it's too positive.  "Remove" is very neutral and implies nothing.  "Lose" implies that it's a good thing to have and its removal is not desired.  "Release" implies that its nothing but a burden and an unwanted one, at that.

Reply:
@marchersimon I think this is the most I have needed your opinion!

Reply:
Do we need a dedicated section like this at all? I thought that the dates next to people's names was enough.

Reply:
That's how it was described in [COMMUNITY-ROLES](https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md#removing-inactive-organization-members).

> Open a PR moving their name to the "Past organization owners" section in MAINTAINERS.md.

Reply:
So we should put both past organization members and owners to the section `Past organization owners`?

Reply:
Oh oops! I've got confused. I'm not sure whether we'd want separate sections or not? Maybe 1 as `Past organization owners`?

Reply:
Let's go with separate sections for now. We can always change it later.

Reply:
@marchersimon could you resolve the conflicts?